ci/build_container/build_container_centos.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+# llvm-5.0.0 from copr
+curl -L -o /etc/yum.repos.d/alonid-llvm-5.0.0-epel-7.repo \
+  https://copr.fedorainfracloud.org/coprs/alonid/llvm-5.0.0/repo/epel-7/alonid-llvm-5.0.0-epel-7.repo
+
+# dependencies for bazel and build_recipes
+yum install -y java-1.8.0-openjdk-devel unzip which \
+               cmake git golang libtool make patch rsync wget \
+               clang-5.0.0 devtoolset-6-libatomic-devel llvm-5.0.0 python-virtualenv
+yum clean all
+
+# latest bazel installer
+BAZEL_VERSION=$(curl -s https://api.github.com/repos/bazelbuild/bazel/releases/latest |
+                  python -c "import json, sys; print json.load(sys.stdin)['tag_name']")
+BAZEL_INSTALLER=bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+curl -OL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/${BAZEL_INSTALLER}
+chmod ug+x ./${BAZEL_INSTALLER}
```

Comment:
Nit: please quote all these string var derefs for whitespace safety.

Reply:
done